resources/sass/app.scss

```diff
@@ -310,12 +310,15 @@ $color-white: #fff;
 }
 
 .ui.four.stackable.cards {
-	.card.bluemarket-projectcard {
-		width: calc(25% - 2.0em);
-		margin-left: 1em;
-		margin-right: 1em;
-		margin-top: 1em;
-		margin-bottom: 1em;
+	margin: 0 auto;
+	margin-top: 10px;
+
+	a.item {
+		all: unset;
```

Comment:
What are we doing this for? This is a code smell. Es preferible resetear las propiedades individuales.